### DISCUSSION TITLE
Switch the formatter.

### DISCUSSION THREAD

**Discussion by semi710** [juspay/nix-dev-home#78] [Ideas]
Switch the formatter from `nixfmt` -> `nixfmt-rfc-style` as this is going to be standard in the upcoming future.

**Comment by srid** [comment]
I'm happy to consider it, but does `nixfmt-rfc-style` still _force_ arguments to appear in _multiple_-lines? Because that would be a showstopper.

**Reply by semi710** [reply]
nop, it does not.

It switches to next line if combined args length exceeds `90-91` characters.

**Reply by srid** [reply]
This is what I observe currently:

<img width="267" alt="image" src="https://github.com/user-attachments/assets/f42f45c5-ffa7-430d-a6b8-171842f83926">

**Reply by semi710** [reply]
Yep, you're right. My editor's formatter was overriding this behavior, and I didn't realize it.

There was a mention of this in the testing thread [here](https://discourse.nixos.org/t/call-for-testing-nix-formatter/39179), but I don't think it will be changed or added as a feature.

I guess sticking with nixfmt is a better option for now ><
